app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1346,9 +1346,10 @@ def get_record_cls(record)
       record.class.base_class.name
     else
       klass = case record
-              when Host, ExtManagementSystem then record.class.base_class
-              when VmOrTemplate then              record.class.base_model
-              else                            record.class
+              when ContainerNode, ContainerGroup, Container then record.class.base_class
+              when Host, ExtManagementSystem                then record.class.base_class
```

Comment:
I'm torn ...

On one hand I like the idea of having the container stuff on its own line to distinguish it from the Infra/Cloud types.  But, on the other hand, I don't like having two cases that end in the same result.

@dclarizio, @Fryguy, @matthewd ... any comments here?